sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/timeExpressions.scala

```diff
@@ -290,3 +294,100 @@ object HourExpressionBuilder extends ExpressionBuilder {
     }
   }
 }
+
+@ExpressionDescription(
+  usage = """
+    _FUNC_([precision]) - Returns the current time at the start of query evaluation.
```

Comment:
It would be nice if you describe the `precision` parameter since this is visible to users. Could you `arguments` like
https://github.com/apache/spark/blob/f4ce0ab8cee043d0b1a8d935fb13ce617d2de873/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/timeExpressions.scala#L136-L141

Reply:
Definitely!